src/server/gc_worker/gc_worker.rs

```diff
@@ -574,9 +574,20 @@ where
 
         self.stats.data.add(&statistics);
 
+        if !latest_version_key.is_empty() {
+            self.delete_raw_write(latest_version_key, raw_modifies);
+        }
+
         Ok(())
     }
 
+    fn delete_raw_write(&mut self, engine_slice: Vec<u8>, raw_modifies: &mut MvccRaw) {
```

Comment:
Why not store `Key` in the variable `latest_version_key`, and pass `Key` into the function? By the way, the word `engine_slice` doesn't make sense to me, especially when it's type is `Vec<u8>` instead of a slice.

Reply:
Change the name to ```delete_raws```

Reply:
> Why not store `Key` in the variable `latest_version_key`, and pass `Key` into the function? By the way, the word `engine_slice` doesn't make sense to me, especially when it's type is `Vec<u8>` instead of a slice.

done